src/localplayer.cpp

```diff
@@ -1057,49 +1060,20 @@ void LocalPlayer::old_move(f32 dtime, Environment *env, f32 pos_max_d,
 
 float LocalPlayer::getSlipFactor(Environment *env, const v3f &speedH)
 {
-
-	if (!touching_ground)
-		return 1.0f;
-
-	float slip_factor = 1.0f;
 	// Slip on slippery nodes
 	const NodeDefManager *nodemgr = env->getGameDef()->ndef();
 	Map *map = &env->getMap();
 	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(
-			floatToInt(getPosition() - v3f(0, 0.05f * BS, 0), BS)));
+			getStandingNodePos()));
```

Comment:
I'm not sure, but wasn't `getStandingNodePos` deprecated, or off by half a node, or something? If it isn't, great, but there was some reason it wasn't used in the original.

Reply:
It's not deprecated.

Reply:
It was touched and extended a few months ago: https://github.com/minetest/minetest/commit/f729b5d2bdc65ce3c51f1004c58cb93a37504b49
Don't worry, I know what I'm doing ;)